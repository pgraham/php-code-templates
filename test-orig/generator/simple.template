This is a sample template with simple substitutions.

This line contains an inline substitution: ${sub1}
This line contains two substitutions: ${sub2} ${sub3}
The following line contains a value on its own
${sub1}
