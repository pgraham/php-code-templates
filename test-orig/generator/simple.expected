This is a sample template with simple substitutions.

This line contains an inline substitution: val1
This line contains two substitutions: val2 val3
The following line contains a value on its own
val1
